.github/workflows/ci.yml

```
@@ -161,7 +161,8 @@ jobs:
       - name: Run clippy
         uses: actions-rs/clippy-check@v1.0.7
         with:
-          name: Clippy (stable)
+          # GitHub displays the clippy job and its results as separate entries
+          name: ${{ jobs.clippy.name }} Results
```

Maybe it doesn't like the variable substitution
```suggestion
          name: Clippy (stable) Results
```